requirements/edx/base.txt

```diff
@@ -1256,7 +1256,7 @@ webob==1.8.7
     #   xblock
 wrapt==1.16.0
     # via -r requirements/edx/paver.txt
-xblock[django]==4.0.1
+xblock[django]==5.1.0
```

Comment:
It’s probably failing due to the references to constraints.txt below, which may no longer be applicable.

Reply:
@robrap , good catch. Ready for re-review.